app/validation/validators.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+
+from app.validation.error_messages import error_messages
+
+from wtforms import validators
+
+
+class IntegerCheck(object):
```

Comment:
Yes, but we can't override their default error, so need to create our own versions of them